src/constants/tokenSafety.tsx

```diff
@@ -0,0 +1,73 @@
+import { Plural, Trans } from '@lingui/macro'
+
+import WarningCache, { TOKEN_LIST_TYPES } from './TokenSafetyLookupTable'
+
+// TODO: Replace this with Steph's article when it is available.
```

Comment:
Is this available?

Reply:
@cartcrom can you own adding this article when it's done?

Reply:
yeah of course, I think i have a tiny jira task for this

Reply:
okay if you have a jira then i will resolve this comment